ivy/functional/frontends/torch/miscellaneous_ops.py

```diff
@@ -92,6 +92,46 @@ def cartesian_prod(*tensors):
     return ret
 
 
+@with_unsupported_dtypes({"2.1.0 and below": "float16"}, "torch")
+@to_ivy_arrays_and_back
+def cdist(x1, x2, p, compute_mode="use_mm_for_euclid_dist_if_necessary"):
```

Comment:
```suggestion
def cdist(x1, x2, p=2.0, compute_mode="use_mm_for_euclid_dist_if_necessary"):
```
following the function signature from the docs https://pytorch.org/docs/stable/generated/torch.cdist.html